tests/test_against_signatory.py

```diff
@@ -0,0 +1,160 @@
+from __future__ import annotations
+
+import jax
+import jax.numpy as jnp
+import pytest
+import signatory
+import torch
+from numpy.random import default_rng
+
+from signax import logsignature, multi_signature_combine, signature
+
+rng = default_rng()
+
+jax.config.update("jax_platform_name", "cpu")
+jax.config.update("jax_enable_x64", True)
+
+
+def test_signature_1d_path():
+    depth = 3
+    length = 100
+    path = rng.standard_normal((length, 1))
+    signature(path, depth)
+
+    path = rng.standard_normal(length)
+    signature(path, depth)
```

Comment:
We can remove this. This actually causes error since `path` is 1D.

Reply:
done!